src/instructlab/configuration.py

```diff
@@ -135,7 +141,11 @@ class _generate(BaseModel):
     # additional fields with defaults
     num_cpus: PositiveInt = DEFAULT_NUM_CPUS
     chunk_word_count: PositiveInt = DEFAULT_CHUNK_WORD_COUNT
-    num_instructions: PositiveInt = DEFAULT_NUM_INSTRUCTIONS
+    # DEPRECATED: see sdg_scale_factor instead
+    # Left in place so that we can still detect and give a warning if its
+    # specified in an old configuraiton file.
+    num_instructions: Optional[int] = -1
```

Comment:
Use Pydantic's Field to mark the entry as deprecated and to exclude it from serialization:

```
from pydantic import Field
```

```suggestion
    num_instructions: Optional[int] = Field(-1, deprecated="see 'sdg_scale_factor' instead", exclude=True)
```

Reply:
I know I tried using this and then gave up, and now I can't remember what the problem was. i don't think I found `exclude=True` though, so that may have been it!

Reply:
Thanks for the tip! This worked great.